cdc/log.cc

```diff
@@ -982,9 +982,9 @@ static bytes get_bytes(const atomic_cell_view& acv) {
     return acv.value().linearize();
 }
 
-static bytes_view get_bytes_view(const atomic_cell_view& acv, std::vector<bytes>& buf) {
+static bytes_view get_bytes_view(const atomic_cell_view& acv, std::forward_list<bytes>& buf) {
```

Comment:
Please file an issue and record it in the commit (with Fixes #), so it can be backported.

Reply:
Done.